database/factories/ModelFactory.php

```diff
@@ -92,7 +92,6 @@
         'evnt_title' => $faker->word(),
         'evnt_subtitle' => $faker->word(),
         'plc_id' => Lara\Section::inRandomOrder()->first()->id,
-        'evnt_show_to_club' => json_encode($faker->randomElements(Lara\Section::all()->pluck('title')->toArray(), $faker->numberBetween(1,2))),
```

Comment:
Did you also adapt the ModelFactory for the new properties?

Reply:
done